app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+        self.check_alive(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+    @staticmethod
+    def check_alive(animal: Animal) -> None:
+        if animal.health <= 0:
+            Animal.alive.remove(animal)
```

Comment:
It's better to add self parameter to the method, static is not a good idea in this case